pnpm-lock.yaml

```diff
@@ -401,7 +401,7 @@ importers:
       merge-stream: ^2.0.0
       pm2: ^5.1.1
       shelljs: ^0.8.4
-      stream-buffers: ^3.0.2
+      streaming-cache: ^0.5.5
```

Comment:
Do we want the carat for streaming-cache? or should we manually keep track of it?

Reply:
We need to pin all of these, which is what #2725 is about.